### DISCUSSION TITLE
Is there a way to disable Magnifier?

### DISCUSSION THREAD

**Discussion by hubbty** [osmandapp/OsmAnd#20276] [General]
I use my phone / Osmand mounted to the handlebar while biking. So there is a lot of motion and vibration under the fingertips. Ever so often when I touch + or - and just want to zoom, this Magnifier thing gets in the way. I'd like to get rid of it alltogether. Is there a way to achieve this?

**Comment by dmpr0** [comment]
Disabling the long tap functionality on the Zoom In / Out button is currently unavailable.

You can try to use two custom buttons with Zoom In / Out actions. Long pressing these custom buttons will only allow you to move them around the screen for placement purposes.

**Reply by hubbty** [reply]
Thank you! 
I'd like to suggest that you better separate the two functions of these buttons on the timescale. My problem would not arise if the Magnifier would kick in only after say one second instead of a tiny fraction of a second.
